src/main/java/com/boardgamesworld/bgrental/boardgame/infrastructure/api/BoardGameController.java

```diff
@@ -0,0 +1,124 @@
+package com.boardgamesworld.bgrental.boardgame.infrastructure.api;
+
+import com.boardgamesworld.bgrental.boardgame.domain.BoardGame;
+import com.boardgamesworld.bgrental.boardgame.domain.BoardGameFacade;
+import com.boardgamesworld.bgrental.boardgame.domain.InvalidBoardGameException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.net.URI;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@RestController
+@RequestMapping("/boardgames")
+public class BoardGameController {
+
+    private BoardGameFacade boardGameFacade;
+    private Logger logger = LoggerFactory.getLogger(BoardGameController.class);
+
+    public BoardGameController(BoardGameFacade boardGameFacade) {
+        this.boardGameFacade = boardGameFacade;
+    }
+
+    @GetMapping
+    public ResponseEntity<List<BoardGameDto>> getAllBoardGames() {
+        List<BoardGame> boardGames = boardGameFacade.getAllBoardGames();
+        List<BoardGameDto> boardGamesDto;
+
+        if (boardGames != null) {
```

Comment:
Lista, która jest nullem nie jest najlepszym pomysłem. Jeśli nie ma żadnych gier, to powinna być pusta lista, co pasuje świetnie w tym przuypadku. 

Jak masz pustą listę zamiast nulla to jest kolejny plus, mozesz leciec forEachem, streamem i wszystkim innym bez obawy że pojawi się NPE